x-pack/legacy/plugins/infra/public/containers/logs/log_highlights/redux_bridges.tsx

```diff
@@ -35,21 +34,8 @@ export const LogHighlightsPositionBridge = withLogPosition(
   }
 );
 
-export const LogHighlightsFilterQueryBridge = withLogFilter(
-  ({ serializedFilterQuery }: { serializedFilterQuery: string | null }) => {
-    const { setFilterQuery } = useContext(LogHighlightsState.Context);
-
-    useEffect(() => {
-      setFilterQuery(serializedFilterQuery);
-    }, [serializedFilterQuery]);
-
-    return null;
-  }
-);
-
-export const LogHighlightsBridge = ({ indexPattern }: { indexPattern: any }) => (
+export const LogHighlightsBridge = () => (
```

Comment:
I guess we can get rid of this component altogether

Reply:
In #50398 we can get rid of it, the log position state still needs the bridge right now though